template/.github/workflows/build.yml.j2

```diff
@@ -258,7 +269,7 @@ jobs:
         with:
           version: v3.13.3
       - name: Set up cargo
-        uses: dtolnay/rust-toolchain@{[ rust_version }]
+        uses: dtolnay/rust-toolchain@be73d7920c329f220ce78e0234b8f96b7ae60248
```

Comment:
note: We might want to include an explicit version here just in case.

Reply:
Ahh I missed that in my "multi-select" :open_mouth:

Reply:
Fixed in 25f1be60eca4f116fc1766a0e4932a34547904b1